forms/jqm/element.js

```diff
@@ -158,6 +166,10 @@ define(function (require) {
 
     },
 
+    onChangeClass: function () {
+      this.$el.attr('class', this.model.attributes['class']);
```

Comment:
Completely possible. jQM will add classes. If this had 2-way data-binding then I'd know that classes coming from the DOM should be backed-up and combined with classes coming from the model. But Mutation Observers seemed a little heavy-duty for this, and without them there's no way to safely back-up the DOM classes (because they could be set at anytime).